internal/controller/drplacementcontrol.go

```diff
@@ -1587,10 +1587,11 @@ func (d *DRPCInstance) generateVRG(dstCluster string, repState rmn.ReplicationSt
 			Name:      d.instance.Name,
 			Namespace: d.vrgNamespace,
 			Annotations: map[string]string{
-				DestinationClusterAnnotationKey: dstCluster,
-				DoNotDeletePVCAnnotation:        d.instance.GetAnnotations()[DoNotDeletePVCAnnotation],
-				DRPCUIDAnnotation:               string(d.instance.UID),
-				rmnutil.IsCGEnabledAnnotation:   d.instance.GetAnnotations()[rmnutil.IsCGEnabledAnnotation],
+				DestinationClusterAnnotationKey:    dstCluster,
+				DoNotDeletePVCAnnotation:           d.instance.GetAnnotations()[DoNotDeletePVCAnnotation],
+				DRPCUIDAnnotation:                  string(d.instance.UID),
+				rmnutil.IsCGEnabledAnnotation:      d.instance.GetAnnotations()[rmnutil.IsCGEnabledAnnotation],
+				rmnutil.UseVolSyncForPVCProtection: d.instance.GetAnnotations()[rmnutil.UseVolSyncForPVCProtection],
```

Comment:
Other annotation have "Annotation" suffix. Maybe rename to "UseVolSyncAnnotation?

Reply:
Ack.